### ISSUE TITLE
Banner isn't showing in 3.0.0

### ISSUE THREAD

**Issue by cikpis** [Daltron/NotificationBanner#258]
Replacing
```swift
private let appWindow: UIWindow? = {
    if #available(iOS 13.0, *) {
        return UIApplication.shared.connectedScenes
            .first { $0.activationState == .foregroundActive }
            .map { $0 as? UIWindowScene }
            .map { $0?.windows.first } ?? UIApplication.shared.delegate?.window ?? nil
    }

    return UIApplication.shared.delegate?.window ?? nil
}()
```
with
```swift
private let appWindow: UIWindow? = {
   return UIApplication.shared.keyWindow
}()
```
solved the problem.

**Comment by kevincador** [comment]
Hi,

I'm having the same kind of problem. The issue is that I have multiple windows in my app that shows overlay things on top of the main window. 

My fix was to get the `last` window instead of the `first`. So:

```private let appWindow: UIWindow? = {
    if #available(iOS 13.0, *) {
        return UIApplication.shared.connectedScenes
            .first { $0.activationState == .foregroundActive }
            .map { $0 as? UIWindowScene }
            .map { $0?.windows.last } ?? UIApplication.shared.delegate?.window ?? nil
    }

    return UIApplication.shared.delegate?.window ?? nil
}()```

I think it's better and will work with more code base.

**Comment by kvkvkvkvkvkv** [comment]
I'm facing this issue too on iOS 13. Suggested solutions to change from "first" to "last" works. Thanks! 
Is this in any PR?

**Comment by JMCPH** [comment]
Has any of you made PR on this issue? Else I will do it.

**Comment by cikpis** [comment]
@JMCPH Feel free to do it.

**Comment by marcheimendinger** [comment]
The fix in merged PR #254 doesn't seem to be present in the latest version of the library. Is it true ?

**Comment by roasted-almond** [comment]
Hi there, it seems this fix is not included in latest library. Shall we reopen the PR?
